python/packages/sdk/serverless_sdk/lib/error_captured_event.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import time
+from builtins import type as builtins_type
+from typing import Optional
+from .tags import Tags
+from .captured_event import CapturedEvent
+from .stack_trace_string import resolve as resolve_stack_trace_string
+
+
+logger = logging.getLogger(__name__)
+
+
+TYPE_MAP = {
+    "unhandled": 1,
+    "handledUser": 2,
+    "handledSdkUser": 3,
+    "handledSdkInternal": 4,
+}
+
+
+def create(
+    error,
+    timestamp: Optional[int] = None,
+    tags: Optional[Tags] = None,
+    type: str = "handledUser",
+    name=None,
+    stack=None,
+):
+    timestamp = timestamp or time.perf_counter_ns()
+    tags = tags or Tags()
+    captured_event = CapturedEvent(
+        "telemetry.error.generated.v1", timestamp=timestamp, custom_tags=tags
+    )
+    _tags = {
+        "type": TYPE_MAP[type],
+    }
+    if isinstance(error, Exception):
+        _tags["name"] = builtins_type(error).__name__
+        _tags["message"] = str(error)
+    else:
+        _tags["name"] = name or builtins_type(error).__name__
+        _tags["message"] = str(error)
+    _tags["stacktrace"] = stack or resolve_stack_trace_string(error)
+    captured_event.tags.update(_tags, prefix="error")
+
+    logger.error(
```

Comment:
It looks as unconditionally logged, while this function might be called from internal error reporter which also logs this log unconditionally.

In Node.js version we fallback to logging only if it's an error explicitly reported by user (and not reported internally by the SDK) https://github.com/serverless/console/blob/430c6d08f603223c05ad71b9b44e58ea4415855d/node/packages/sdk/lib/create-error-captured-event.js#L40-L45

Reply:
Good catch, I've amended it 🙏